extensions/wikia/AppPromoLanding/AppPromoLandingController.class.php

```diff
@@ -72,7 +77,7 @@ public function index() {
 		// Inject the JS
 		$srcs = AssetsManager::getInstance()->getGroupCommonURL( 'app_promo_landing_js' );
 		foreach( $srcs as $src ) {
-			$this->wg->Out->addScript( "<script type=\"{$wgJsMimeType}\" src=\"{$src}\"></script>" );
+			$this->wg->Out->addScript( "<script type=\"{$this->wg->JsMimeType}\" src=\"{$src}\"></script>" );
```

Comment:
Cool beans. Will switch to Wikia::addAssetsToOutput() to output the whole group.